net/peer_tracker.go

```diff
@@ -57,15 +127,55 @@ func (tracker *PeerTracker) List() []*types.ChainInfo {
 	return out
 }
 
+// ListTrusted returns the chain info of the trusted tracked peers. The info tracked by the tracker can
```

Comment:
Unsure if the below methods `ListTrusted` and `TrustedPeers` should exists or if `Peers` and `List` should accept parameters, thoughts?

Reply:
I don't see any reason for those methods to exist, except as internal helpers, with or without a parameter. They are not "interface" methods. 

Don't test them directly, just test the behaviour we really care about - that the non-trusted peers are not included in the trusted head calculation.